### ISSUE TITLE
Show Builder - additional information for giveaways

### ISSUE THREAD

**Issue by seankwilliams** [codefordenver/Comrad#348]
depends on #347

Additional information for giveaways will be displayed when a user expands a giveaway item, including a form for entering the winner's information

In addition to the copy, display:
* List out any custom properties that the traffic events have
* Below that, have a button for "Enter Winner Information"
  * Clicking the Enter Winner Information opens a modal window with the following fields:
    * No Callers (checkbox)
    * Winner's Name (textbox)
    * Winner's Phone Number (textbox)
    * Winner's Email (textbox)
    * Winner's Address (textbox)
    * Mark if the winner is a KGNU listener-member (checkbox)
    * Email the ticket information to the user (checkbox, checked by default)
  * Saving the form should: 
    * save the information to the giveaway instance in the database
    * Trigger an email to the winner if the checkbox was checked
    * Trigger an email to the station's giveaway manager (a configurable email address)
  * After the winner's info has been saved, the giveaway button should be changed to "View Winner Information", and should display the values from the original form in a modal when clicked. 
    * The values should not be editable once they are saved once.

**Comment by seankwilliams** [comment]
- [ ] display copy and custom properties in SHow Builder
- [ ] Enter winner information button 
- [ ] Have button open a modal
- [ ] Form in the modal
- [ ] Custom field support on the form
- [ ] Save data to database
- [ ] Email trigger to the winner
- [ ] Email trigger to the giveaway manager
- [ ] View Winner Info

**Comment by seankwilliams** [comment]
On hold while KGNU considers using Jotform for the Giveaway form instead. With Jotform, they could do some additional automations, like adding winner info to a spreadsheet. This actually might be a better solution for stations as well because it lets them customize the actions that happen when winner's info is submitted.
